### DISCUSSION TITLE
是否有开发鸿蒙操作系统（HarmoneyOS）的计划？

### DISCUSSION THREAD

**Discussion by Callmewxh** [localsend/localsend#1939] [L-简体中文]
期待有适用于鸿蒙系统的版本。

**Comment by xiaojun207** [comment]
这应该取决于flutter是否支持，让google的flutter支持鸿蒙，可能性比较小

**Comment by cmlanche** [comment]
闪电藤已经完成了纯血鸿蒙的适配，在准备软著和备案，2周左右就可以上线了，和localsend可以配合使用

**Reply by TiraMisuLab01** [reply]
现在闪电藤已经在鸿蒙next商店上线了😎

**Reply by KyleChang-zkh** [reply]
试了下闪电藤，没法跟ios设备localsend互传。。 是有bug吗

**Comment by xihale** [comment]


**Reply by cmlanche** [reply]
> 框架适配就可以编译了。

已经上线了

**Comment by azhu003** [comment]
实现了一个HarmonyOS Next版本的LocalSend 欢迎试用👏🏻 https://github.com/azhu003/localsend-harmony.git
